frontend/components/ui/login-card.tsx

```
@@ -0,0 +1,118 @@
+        }
+    };
+
+    return (
+        <section className='w-full md:w-[92%] lg:w-3/5 xl:w-[45%]'>
+            <div className='singUp-wrap'>
+                <h2 className='title'>Welcome back!</h2>
+                <p className='tracking-tight text-body-color'>
+                    Hey there! Ready to log in? Just enter your username and password below and you&apos;ll be back in
+                    action in no time. Let&apos;s go!
+                </p>
+                <button className='account__social-btn'>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)